src/gl/lineFeature.js

```diff
@@ -225,22 +240,17 @@ geo.gl.lineFeature = function (arg) {
 
     i = 0;
     k = 0;
+    var order = m_this.featureVertices();
```

Comment:
just to be consistent, can we move the declaration at the beginning? Also, same for the addVert method as well.